src/app/app.module.ts

```diff
@@ -3,10 +3,13 @@ import { NgModule } from '@angular/core';
 
 import { AppComponent } from './app.component';
 import { LandingPageModule } from './landing-page/landing-page.module';
+import { ApiComponent } from './api/api.component';
+import { AppRoutingModule } from './app-routing.module';
+import { HttpClientModule } from '@angular/common/http';
 
 @NgModule({
-  declarations: [AppComponent],
-  imports: [BrowserModule, LandingPageModule],
+  declarations: [AppComponent, ApiComponent],
```

Comment:
ApiComponent should have its own module and it should be imported here. Similar to LandingPageModule

Reply:
Done